kitchen/src/main/java/nl/teamtwee/bep3/restaurant/kitchen/infrastructure/driver/web/request/UpdateOrderRequest.java

```diff
@@ -0,0 +1,5 @@
+package nl.teamtwee.bep3.restaurant.kitchen.infrastructure.driver.web.request;
+
+public class UpdateOrderRequest {
+
```

Comment:
Verwachten we een request waar niks in zit?

Reply:
currently the update has no functionality, so not yet implemented, i suggest we merge anyway to get all of the services' base functionality together in one  place